reeds/modules/do_RE_EDS_sOptimisation.py

```diff
@@ -34,8 +34,8 @@ def do(out_root_dir: str, in_simSystem: fM.System, in_template_imd: str = None,
        state_physical_occurrence_potential_threshold:List[float]=None,
        state_undersampling_occurrence_potential_threshold: List[float]=None,
        undersampling_fraction_threshold:float=0.9,
-       equil_runs: int = 1, steps_between_trials: int = 50, trials_per_run: int = 12500,
-       non_ligand_residues: list = [],
+       equil_runs: int = 1, prod_runs = 1, steps_between_trials: int = 50,
```

Comment:
@eruijsena Here I would maybe change the default to `prod_runs = 9` so the complete default would be to have a 10% eq. followed by 90% prod. 

Its something I wanted to do with the general options but unfortunately haven't had the time to do.